lib/purchase.tsx

```diff
@@ -8,7 +8,7 @@ const purchase = async (signer: Signer, initialData?: string) => {
 
   const contract = new ethers.Contract(contractAddress, abi, signer)
   try {
-    const tx = await contract.purchase(1, initialData)
+    const tx = await contract.purchase(1, initialData, { value: ethers.utils.parseEther("0.001") })
```

Comment:
we should set the value to a env variable

Reply:
I think we should read it from the contract.

value can change, at contract level, so we need to read it from there instead of hard-coding as ENV variable. 
do you agree?

will update now. ✏️

Reply:
updated to read price from contract 👍 

<img width="772" alt="Screenshot 2023-03-01 at 1 43 10 PM" src="https://user-images.githubusercontent.com/23249402/222205421-4e8562bb-29bf-4308-bd76-65ecc2cc7591.png">